app/src/main/java/org/piramalswasthya/cho/ui/register_patient_activity/patient_details/PatientDetailsFragment.kt

```
@@ -175,113 +171,50 @@ class PatientDetailsFragment : Fragment() , NavigationAdapter {
-                            // Clean up detectoar
-                            faceDetector.close()
+                // Directly show the captured photo (without MediaPipe-based face detection)
+                Glide.with(this).load(photoURI).placeholder(R.drawable.ic_person).circleCrop()
+                    .into(binding.ivImgCapture)
 
-                            // Get face embeddings
-                            embeddings = faceNetModel.getFaceEmbedding(faceBitmap)
+                // Generate embeddings on the full image (best-effort without explicit face cropping)
+                embeddings = try {
+                    faceNetModel.getFaceEmbedding(imageBitmap)
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)